index.html

```diff
@@ -0,0 +1,129 @@
+                            <img id="closeMenuImage" src="images/xMenu_50.png" alt="Close Menu">
+                        </a>
+                        <a id="home" href="#home" tabindex="1">Home</a>
+                        <a id="about" href="#about" tabindex="1">About</a>
+                        <a href="#Program" tabindex="2">Program</a>
+                        <a href="#Join" tabindex="3">Join</a>
+                        <a href="#Sponsor" tabindex="4">Sponsor</a>
+                        <a href="#News" tabindex="5">News</a>
+                    </div>
+                    <div class="importantLink">
+                        <a href="#contact" tabindex="5">CC Campaign</a>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <section>
+            <div id="main" class="generalPadding main">
+                <!--Background image-->
+                <div  id="mainBackground" class="mainBackground"></div>
+                <!--Menu Hamburger-->
+                <div id="menuHamburgerContainer" class="menuHamburgerContainer">
+                    <a id="menuHamburger" href="javascript:void(0);" class="menuHamburger">
+                        <img id="baraMenuImage" src="images/barMenu.png" alt="Hamburger Menu" class="">
+                    </a>
+                </div>
+                <div class="mainContainer">
+                    <h2>"Hello! Sharing world"</h2>
+                    <h1>CREATIVE COMMONS<br>GLOBAL SUMMIT<br>2015</h1>
+                    <p>A joyful celebration believing in the value of openess and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea</p>
+                    <h3>2015.10.15&#40;THU&#41; ~ 16&#40;FRI&#41;</h3>
+                    <H4>&#64; National Museum Korea, Art Center Nabi and more</H4>
+                    <!-- <img src="https://img.freepik.com/premium-vector/random-vinyl-records-pattern-music-illustration-creative-luxury-cover_510351-12718.jpg?w=2000" alt="Red Pattern"> -->
+                </div>
+            </div>
+        </section>
+        <section class="kinds generalPadding">
+            <div class="kindsTitles">
+                <h1>Main Program</h1>
+                <hr>
+            </div>
+            <div id="kindsContainer" class="kindsContainer">
+
+            </div>
+            <div class="kindsLkn">
+                <a href="javascript:void(0);">SEE THE WHOLE PROGRAM</a>
+            </div>
+            <div class="kindsBtn">
+                <a href="javascript:void(0);" class="">Join CC Global summit 2015</a>
+            </div>
+        </section>
+        <section class="bands generalPadding">
+            <div class="bandsTitles">
+                <h1>Featured Music Bands</h1>
+                <hr>
+            </div>
+            <div class="bandsContainer" id="bandsContainer">
+
+            </div>
```

Comment:
Great work so far on implementing this section, but according to the design kindly I suggest you stick to the design as much as possible to make your project look more professional by following the same `font` `margin/padding`, `color` etc.

> ## The required design
>
><img width="1440" alt="Screenshot 2023-07-02 at 12 23 17 AM" src="https://github.com/alexansaa/ConcertAdv/assets/91237525/17baf8bc-0d9e-4509-8c12-9b8e58451acb">
>
> ## Your Design
>
><img width="1440" alt="Screenshot 2023-07-02 at 12 09 03 AM" src="https://github.com/alexansaa/ConcertAdv/assets/91237525/7cdc0493-0faf-4c49-b0a3-2d1abdf5ea53">